token/cli/src/main.rs

```diff
@@ -3264,19 +3291,24 @@ mod tests {
             &multisig_member_help,
         )
         .get_matches_from(args);
-        let (sub_command, sub_matches) = app_matches.subcommand();
+        let (sub_command_, sub_matches) = app_matches.subcommand();
+        let sub_command = CommandName::from_str(sub_command_).unwrap();
```

Comment:
nittiest of nits: in Rust, you can rebind an old variable name, so no need to have the underscore.  You can do
```
 let (sub_command, sub_matches) = app_matches.subcommand();
let sub_command = CommandName::from_str(sub_command).unwrap();
```

Reply:
Didn't knew this, cool :)